projects/rulenode-core-config/src/lib/components/action/check-point-config.component.html

```diff
@@ -1,7 +1 @@
-<section [formGroup]="checkPointConfigForm" fxLayout="column">
-  <tb-queue-autocomplete
-    required
-    [queueType]="serviceType"
-    formControlName="queueName">
-  </tb-queue-autocomplete>
-</section>
+<section [formGroup]="checkPointConfigForm"></section>
```

Comment:
This rule node config component is not necessary. You need to delete this component and use component tb-node-empty-config as the example entity-type-switch rule node.